connectors/src/connectors/zendesk/lib/node-zendesk-types.d.ts

```diff
@@ -2,7 +2,7 @@ import "node-zendesk";
 
 import type { ZendeskClientOptions } from "node-zendesk";
 
-interface Brand {
+interface ZendeskFetchedBrand {
```

Comment:
Noob question: any reason we have a declaration file for zendesk and not the other connectors?

Reply:
This is because we are using an external library that wraps the API instead of defining our own (typed) wrapper

Reply:
And also the wrapping library's types are a bit broken